portfolio/src/main/webapp/index.html

```diff
@@ -29,5 +29,14 @@ <h1>General Nana Baafi</h1>
       <img src="/images/prom_picture.jpg" width="225" height="300" />
       <img src="/images/first_snow.jpg" width="300" height="300" />
     </div>
+    <div id="content">
+        <b>Have Questions? Let's have a chat!</b>
+        <form onsubmit="analyzeMessage(); return false">
```

Comment:
let's use JS to add event listeners here, especially for functions with multiple statements.  use addEventListener().